openpype/hosts/houdini/api/lib.py

```diff
@@ -623,11 +620,6 @@ def get_frame_data(node, handle_start=0, handle_end=0, log=None):
         data["frameEndHandle"] = int(node.evalParm("f2"))
         data["byFrameStep"] = node.evalParm("f3")
 
-    data["handleStart"] = handle_start
-    data["handleEnd"] = handle_end
-    data["frameStart"] = data["frameStartHandle"] + data["handleStart"]
-    data["frameEnd"] = data["frameEndHandle"] - data["handleEnd"]
```

Comment:
There are some mess! 
I think setting start and end to dull range by default may help avoid some frustration! 
> I'm a little confused tbh, e.g. collect_review_data is expecting the full range and I don't if burnin and other post processing scripts expects which data to be exists.  
```suggestion 
    data["frameStart"] = data["frameStartHandle"] 
    data["frameEnd"] = data["frameEndHandle"]
```